less/less.d.ts

```diff
@@ -72,6 +77,8 @@ declare namespace Less {
 }
 
 interface LessStatic {
+    options: Less.StaticOptions;
```

Comment:
i haven't used this before, but the docs seems to indicate setting `less = <options>`, not `less.options = <options>` http://lesscss.org/usage/#using-less-in-the-browser-options